tasks/discover-granules/package.json

```diff
@@ -41,19 +41,23 @@
     "@cumulus/cumulus-message-adapter-js": "^1.1.0",
     "@cumulus/ingest": "1.19.0",
     "@cumulus/logger": "1.19.0",
+    "@cumulus/common": "1.19.0",
+    "@cumulus/aws-client": "1.19.0",
```

Comment:
Where does discover-granules directly depend on these packages?

Reply:
Hmm, it doesn't.   The blame on that line is a merge commit from master... let me take a quick look

Reply:
Ah, common/client should have been moved back to test.   💯  good catch.